### PR TITLE
Add a bit of caching to speed up the chart

### PR DESCRIPTION
**Pull request by zestyping**
Some low-hanging fruit to speed up the chart a bit, as it's getting slow.  (Still, the largest speed gains will come from avoiding rendering tables with very large numbers of cells.)

### CODE REVIEW

**Comment by zestyping** [comment]
Ready to review and merge.  Optional, but low-risk and should help us a bit.

File: app/src/main/assets/chart.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
         {% for column in columns %}
           <th class="{{column.start == nowColumnStart ? 'now' : ''}}" scope="col">
             {{column.headingHtml | raw}}
-          </th>
+                                                                                         </th>
```
Comment (llvasconcellos) [inline app/src/main/assets/chart.html:38]: @zestyping Did you put this extra spaces on purpose?

**Comment by llvasconcellos** [comment]
Nothing to add. It's tested and running. Didn't notice performance improvement though.

**Comment by llvasconcellos** [comment]
The first merge was out of order. I should have merged https://github.com/projectbuendia/client/pull/35 first.

**Pull request merged**